### DISCUSSION TITLE
Problems encountered in the use of isce2 and autoRIFT

### DISCUSSION THREAD

**Discussion by mintaizu** [isce-framework/isce2#755] [Q&A]
Ok, I am going to use two Sentinel 1A data for offset tracking. The tool I use is autoRIFT. According to your previous comments and the instruction file of autoRIFT, I know that I need to use topsApp.py to preprocess the sentinel data, generate slc files and then use autoRIFT, but I use topsApp.py and No slc file was generated. So I tried to use the stacksentinel.py script for processing. stacksentinel.py did generate two slc.full files for co-registration, but the .mat file generated by offset tracking of slc.full always shows NAN values, this is confusing me. I'm currently stuck at this step, can you give me some advice? Thank you again!

**Comment by EJFielding** [comment]
I learned that there is a special stacksentinel workflow for creating the SLC images you need for autoRIFT from a stack of Sentinel-1 images. You have to use the `-W offset` option when you setup your run files.
